scripts/build-website.mjs

```diff
@@ -25,7 +26,7 @@ const runYarn = (command, args, options) =>
 const IS_PULL_REQUEST = process.env.PULL_REQUEST === "true";
 const PRETTIER_DIR = IS_PULL_REQUEST
   ? DIST_DIR
-  : path.dirname(require.resolve("prettier"));
+  : url.fileURLToPath(new URL("../node_modules/prettier", import.meta.url));
```

Comment:
Can we use `import-meta-resolve` package here?

Reply:
Problem is not about resolve, `require.resolve` resolves to dev version, don't know why. But it will resolve to stable version if I remove `exports`.

Reply:
I got it.

Reply:
https://nodejs.org/api/packages.html#self-referencing-a-package-using-its-name